config/app_config.cfg

```diff
@@ -9,6 +9,8 @@ DPNAME = ngic-skx2
 MCC = 000
 MNC = 000
 TAC = 0
+NUM_IMSIs = 2
+IMSIs = 208014567891200 208014567891201
```

Comment:
can you pleae give example how this looks like if we need 100 IMSIs ?

Do we have to list IMSIs here or we need to give fromIMSI-toIMSI ?

Reply:
Typically we may add 100's of IMSIs. It may not good idea to list all IMSI's. 

Just giving common-prefix should be good and then that way search also becomes simple. We dont need to go through IMSI's sequentially.

Reply:
you are right, I didnt put this point into account,
I think it is good to make it for example: IMSIs = 208014567891200:2080145678912100
which means select this DP if the current IMSI is included in this range [208014567891200:2080145678912100]

Also we can make it possible to add multiple ranges, for example IMSIs = 208014567891200:2080145678912100 208014567895300:2080145678915700
I can start working on it if you agree!

Reply:
208014567895400:208014567891700. << Looks some type in this range

Reply:
@ajamshed @krsna1729  What is your opinion? I  agree with approach of giving IMSI based selection. And selection criteria to be range based. Please comment on this proposal.

Reply:
Using the IMSI prefix is the most common way to do a range from what I have seen.  I have also seen '-' range using the IMSI prefix, e.g. 301-304 for mobile network codes.  The question would be what, if anything, should be done for multiple noncontiguous ranges.

Reply:
Yes true. Putting just prefix  from-to/. like 301-304 would be great. this means that ignore other digits in the IMSI.  Also we should be able to provide multiple prefix ranges like this. Just like the way we give NUM Selection rules.

Reply:
the selection based on IMSI came to my mind when we faced a use case where we needed only few UEs to connect to specific DP(sounds strange but yeah), and what if we need only some IMSIs that belongs to a mobile network codes and not all its IMSIs.

Reply:
Hello guys,The code works just fine and builds successfully, but SPGWC gives id 0 to SPGWU (two pics below show logs of spgwc and cpiface container, I am not sure where is exactly the issue, since my code runs only when a UE tries to connect.

Could anyone help me to figure out the issue. I am still not familiar with the OMEC code. I appreciate if you could help me also with any tips to debug the code.

This will help me a lot to contribute in the future and implement this feature https://github.com/omec-project/ngic-rtc/pull/134/commits/591862483980e50863e51b71efb6655727093d35#r441407671

![spgwc_error](https://user-images.githubusercontent.com/25632982/91669040-c9a71b80-eadf-11ea-882b-b25526bb4364.png)

![cpiface_error](https://user-images.githubusercontent.com/25632982/91669057-e2173600-eadf-11ea-9a04-2b847f2ca157.png)

Reply:
@hyunsun any help on this please ?!

Reply:
When DP connects to CP, DP pass the DP_name to CP.  CP uses this DP_name to find the corresponding DP_ID. DP_Name to DP_ID is config and present in app_config.cfg.

Reply:
When DP connects to CP, DP pass the DP_name to CP.  CP uses this DP_name to find the corresponding DP_ID. DP_Name to DP_ID is config and present in app_config.cfg.

Reply:
Sorry thakurajayL, I didnt notice the comment, Actually I found that injecting the IMSIs to dpInfo object overwrite the dpName field. I m not sure why, but what i did is injecting the IMSIs first before assigning the dpName. Do you have any idea what is the issue on this !

Reply:
Can you point to the code. I still see dp_name assigned before IMSIs.